### DISCUSSION TITLE
Link field in forms [Aug 18]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6235] [Discord Threads]
> 🙋‍♂️ **knicklicht4421**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Link fields can be used in forms. Both has-many and many-to-many. Was that your query about?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1131102317555486803)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
